From 524e97c8c2149fc5bb0e907fe51e18df4cd0f1c0 Mon Sep 17 00:00:00 2001
From: Subhankar Paul <subhankar.paul@intel.com>
Date: Mon, 25 May 2015 14:28:02 +0530
Subject: [PATCH] ASoC: Intel: [revert me] Increasing SST ipc timeout to 2sec.

There is a system performance issue for which wait-queue is taking long enough
time to execute, and causing timeout in sst driver which breaks the use cases.

Till performance issue get fixed in CHT, timeout will be increased to 2sec.
This should get reverted once system performance issue fixed.

Change-Id: I6d8fe961f244a865fcb72cf1bd2b6b2c03f5dbf7
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-36346
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-10464
Signed-off-by: Subhankar Paul <subhankar.paul@intel.com>
---
 sound/soc/intel/sst/sst.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/sst/sst.h b/sound/soc/intel/sst/sst.h
index 691b96348e34..f1474bb7f029 100644
--- a/sound/soc/intel/sst/sst.h
+++ b/sound/soc/intel/sst/sst.h
@@ -71,7 +71,7 @@ enum sst_algo_ops {
 	SST_GET_ALGO = 1,
 };
 
-#define SST_BLOCK_TIMEOUT	1000
+#define SST_BLOCK_TIMEOUT	2000
 
 /* SST register map */
 #define SST_CSR			0x00
-- 
1.9.1

